src/quo2/components/code/snippet_preview/view.cljs

```diff
@@ -0,0 +1,10 @@
+(ns quo2.components.code.snippet-preview.view
+  (:require [quo2.components.code.common.view :as code-common]))
+
+(defn view
+  [{:keys [language]} children]
+  [code-common/view
+   {:preview?  true
+    :language  language
+    :max-lines 0}
+   children])
```

Comment:
This one should pass `:theme` as `:dark` always, since this variant is only available in dark mode. But if we are not using the `theme` key in the common component, it'll have no effect, we can address it now or in the follow up I mentioned before